.github/workflows/release_latest_version.yml

```diff
@@ -170,3 +170,29 @@ jobs:
 
       - name: tag-and-push-images
         run: ./ops/cli.py tag-and-push-images
+
+      - name: Read Version Data
+        id: version
+        run: |
+          echo "::set-output name=VERSION::v$(cat version)"
+          echo "::set-output name=CHANGELOG::$(sed -n '1d;/$#/!p;//q' CHANGELOG.md)"
+
+      - name: Deploy Docs
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          personal_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: derived/docs/site
+          keep_files: true
+          destination_dir: ${{ steps.version.outputs.VERSION }}
+
+      - name: Add Release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ steps.version.outputs.VERSION }}
+          body: |
+            Docs available at [broadinstitute.github.io/wfl/${{ steps.version.outputs.VERSION }}](https://broadinstitute.github.io/wfl/${{ steps.version.outputs.VERSION }})
+
+            [Changes](https://github.com/broadinstitute/wfl/blob/${{ steps.version.outputs.VERSION }}/CHANGELOG.md):
+            ${{ steps.version.outputs.CHANGELOG }}
```

Comment:
My understanding is this step of Action will create a separate git tag for the docs, in addition to what `make_git_tag` in `cli.py` created?

Reply:
The `create-release` action here only makes a tag if one doesn't already exist (see how `commitish` [here](https://github.com/actions/create-release#inputs) is unused if the tag already exists). The action here is like if we went to [the releases page ourselves](https://github.com/broadinstitute/wfl/releases/new) and made a release from an existing tag.

Tags are already present on the [releases page](https://github.com/broadinstitute/wfl/releases) anyway, this just formalizes it so we can add some description text

Reply:
I see. Thanks for the explanation!